esmvaltool/recipes/recipe_climwip.yml

```
@@ -0,0 +1,116 @@
+# ESMValTool
+# recipe_climwip.yml
+---
+documentation:
+  description: EUCP ClimWIP
+
+  authors:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+    - camphuijsen_jaro
+
+  maintainer:
+    - kalverla_peter
+    - smeets_stef
+    - brunner_lukas
+
+  references:
+    - brunner2019
+    - lorenz2018
+    - knutti2017
+
+datasets: &model_data
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: "r(1:4)i1p1"}
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+obs_data: &obs_data  # for climwip performance metrics
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3}
+
+preprocessors:
+  climwip_general: &general
+    regrid:
+      target_grid: 2.5x2.5
+      scheme: linear
+    mask_landsea:
+      mask_out: sea
+    extract_region:
+      start_longitude: 0.0
+      end_longitude: 39.0
+      start_latitude: 30.0
+      end_latitude: 76.25
+
+  climatological_mean:
+    <<: *general
+    climate_statistics:
+      operator: mean
+
+  detrended_std:
+    <<: *general
+    detrend:
+      dimension: time
+      method: linear
+    climate_statistics:
+      operator: std_dev
+
+  temperature_anomalies:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+
+diagnostics:
+  calculate_weights_climwip:
+    variables:
+      tas:
+        start_year: 1995
+        end_year: 2014
+        mip: Amon
+        preprocessor: detrended_std
```

```suggestion
        preprocessor: climatological_mean
```

I hadn't suggested that change yesterday but based on my @ruthlorenz and my comments I would now argue that we go with only climatology-based metrics for now?!

---

I was just doing that ;-)